docs/package.json

```diff
@@ -13,9 +13,13 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "npm run build:api && npm run build:md-magic && npm run hugo",
+		"build:multi-version": "npm run build:api && npm run build:api:multi-version && npm run build:md-magic && npm run hugo",
 		"build:api": "npm run build:api-rollup && npm run build:api-documentation",
+		"build:api:multi-version": "npm run build:api-rollup:multi-version && npm run build:api-documentation:multi-version",
```

Comment:
I'm currently running `npm run build:md-magic` under `build:multi-version`. Would it be repeated if I run it on both, or should I move the call to `build:api:multi-version`?

Reply:
Sorry, I meant you should manually run the script and push the changes to the branch. You added new scripts to the package.json, so it will generate changes to the `scripts` table in the docs README.